### PR TITLE
refactor: add LockUtils

### PR DESCRIPTION
**Pull request by classicalliu**
Using LockUtils to deal with lock/lockHash/address/blake160 convert

### CODE REVIEW

File: packages/neuron-wallet/src/utils/lockUtils.ts [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+export default class LockUtils {
+  static systemScriptInfo: { codeHash: string; outPoint: OutPoint } | undefined
+
+  static async systemScript() {
+    if (this.systemScriptInfo) {
+      return this.systemScriptInfo
+    }
+
+    const genesisHash: string = await core.rpc.getBlockHash('0')
+    const genesisBlock = await core.rpc.getBlock(genesisHash)
```
Comment (ashchan) [inline packages/neuron-wallet/src/utils/lockUtils.ts]: We should use `getBlockByNumber('0')` to reduce one RPC call.

File: packages/neuron-wallet/src/utils/lockUtils.ts [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    if (typeof data === 'string') {
+      blake2b.update(core.utils.hexToBytes(data))
+    } else {
+      // if Uint8Array
+      blake2b.update(data)
+    }
+    const codeHash: string = blake2b.digest('hex')
+    const cellOutPoint: CellOutPoint = {
+      txHash: systemScriptTx.hash,
+      index: '0',
```
Comment (ashchan) [inline packages/neuron-wallet/src/utils/lockUtils.ts]: If I recall correctly the index should be `1` instead?

File: packages/neuron-wallet/src/utils/lockUtils.ts (hunk 1 above)
Comment (classicalliu) [reply packages/neuron-wallet/src/utils/lockUtils.ts]: updated, thanks

File: packages/neuron-wallet/src/utils/lockUtils.ts (hunk 2 above)
Comment (classicalliu) [reply packages/neuron-wallet/src/utils/lockUtils.ts]: updated

**Comment by Keith-CY** [comment]
conflicts

**Pull request merged**